src/scripts/tidy5e-npc.js

```diff
@@ -30,10 +30,10 @@ export default class Tidy5eNPC extends dnd5e.applications.actor
    * @return {Object}
    */
   static get defaultOptions() {
-    let defaultTab = game.settings.get("tidy5e-sheet", "defaultActionsTab") != 'default' 
+    let defaultTab = game.settings.get("tidy5e-sheet", "defaultActionsTab") != 'default'
```

Comment:
Don't add different end of line sequences

Reply:
This isn't an EOL change, my IDE strips trailing whitespace (same with tabulation below). I thought I'd skipped them when committing but it seems some slipped through.